hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java

```diff
@@ -485,4 +497,12 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
                         txnEstimateFactory)
                 .usageGiven(fromPbj(body), sigValueObj, token));
     }
+
+    private boolean checkForZeroAccountIfContractCall(boolean isContractCall, TokenUpdateTransactionBody op) {
+        return isContractCall || !isZeroAccount(op.treasury());
+    }
+
+    private boolean isZeroAccount(@NonNull AccountID accountID) {
+        return accountID.equals(AccountID.newBuilder().accountNum(0L).build());
```

Comment:
Maybe have a private static final `ZERO_ACCOUNT_ID`.  Probably doesn't make a difference in the long run, performance-wise.  Might be easier to read.

Reply:
Done